calico-cloud_versioned_docs/version-18/networking/egress/egress-gateway-azure.mdx

```diff
@@ -254,7 +252,7 @@ spec:
 EOF
 ```
 
-Even with BGP filter, {{prodname}} still advertises pod IP blocks. To prevent {{prodname}} from advertising pod IP blocks, it is possible to set
+Even with BGP filter, {{prodname}} still advertise pod IP blocks. To prevent {{prodname}} advertising pod IP blocks, it is possible to set
```

Comment:
```suggestion
Even with the BGP filter, {{prodname}} still advertises pod IP blocks. To prevent {{prodname}} advertising pod IP blocks, it is possible to set
```